api/src/api/routes.ts

```diff
@@ -75,6 +78,18 @@ api.post('/initialise/', async (req, res) => {
   res.json({success: true});
 });
 
+/**
+ * Handle info requests, basic identifying information for this server
+ */
+api.get('/info', async (req, res) => {
+  const info = {
+    name: CONDUCTOR_INSTANCE_NAME,
+    url: CONDUCTOR_PUBLIC_URL,
+    description: CONDUCTOR_DESCRIPTION,
+  };
```

Comment:
Can you type this interface in the data models package so both app and API can reference it?

Reply:
Modified slightly to fit the existing ListingInformation type.

Reply:
See my commit to address this - my comment was to suggest that the API should explicitly annotate the model that is being returned which is shared between the App and API. Currently, someone could come along and change 'name' to 'label' and there would be no warning anywhere.

Reply:
Sorry realise you hadn't committed your updates yet. I can revert if needed.